doc/glm/second_level_model.rst

```diff
@@ -0,0 +1,105 @@
+.. _second_level_model:
+
+===================
+Second level models
+===================
+
+.. topic:: **Page summary**
+
+   Second level models are used to take fMRI analysis to the group level. Once individual subject data has been processed in a common space (e.g. MNI, Talairach, or subject average), the data can be grouped and statistical tests  performed to make broader inferences on fMRI activity.
```

Comment:
We should say that this page focuses on running a second-level model with nilearn (and make sure that the page is indeed framed this way).